our_files/custom.js

```diff
@@ -1764,66 +1764,161 @@ Mobile Outgoing Markup:
+    <div class="detailItemTable borderSection bcolor-s4 bcolor" id="detailItemTabledetailItemsDiv00">
+      <div class="detailChildRecord border-v" id="childRecorddetailItemsDiv00_0">
+          ...
+          <div class="detailChildField field">
+            <div class="detailChildFieldLabel label text-h5 detailItemsTable_ITYPE">Material Type</div>
+            <div class="detailChildFieldValue fieldValue text-p detailItemsTable_ITYPE">Permanent Reserve</div>
+          </div>
+          ...
+          <div class="detailChildField field">
+            <div class="detailChildFieldLabel label text-h5 detailItemsTable_SD_ITEM_HOLD_LINK">Request Item</div>
+            <div class="detailChildFieldValue fieldValue text-p detailItemsTable_SD_ITEM_HOLD_LINK">
+              <div 
+                class="asyncFieldSD_ITEM_HOLD_LINK asyncInProgressSD_ITEM_HOLD_LINK" 
+                id="asyncFielddetailItemsDiv0SD_ITEM_HOLD_LINK31518200027229">Unavailable</div>
+              ...
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+Mobile Outgoing Markup: 
+    <div class="detailItems ">
+    <div class="detailItemTable borderSection bcolor-s4 bcolor" id="detailItemTabledetailItemsDiv00">
+      <div class="detailChildRecord border-v" id="childRecorddetailItemsDiv00_0">
+          ...
+          <div class="detailChildField field">
+            <div class="detailChildFieldLabel label text-h5 detailItemsTable_ITYPE">Material Type</div>
+            <div class="detailChildFieldValue fieldValue text-p detailItemsTable_ITYPE">Permanent Reserve</div>
+          </div>
+          ...
+          <div class="detailChildField field">
+            <div class="detailChildFieldLabel label text-h5 detailItemsTable_SD_ITEM_HOLD_LINK">Request Item</div>
+            <div class="detailChildFieldValue fieldValue text-p detailItemsTable_SD_ITEM_HOLD_LINK">
+              <div 
+                class="asyncFieldSD_ITEM_HOLD_LINK asyncInProgressSD_ITEM_HOLD_LINK" 
+                id="asyncFielddetailItemsDiv0SD_ITEM_HOLD_LINK31518200027229">Available</div>
+              ...
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+*/
 var deUnavailablePermReserve = function() {
-  $J(".asyncFieldSD_ITEM_HOLD_LINK")
-    .not(".hidden")
-    .ajaxComplete(function() {
-      $J(".asyncFieldSD_ITEM_HOLD_LINK")
-        .not(".hidden")
-        .each(function(i, elem) {
-          var materialText = $J(elem)
-            .closest("tr")
-            .find(".detailItemsTable_ITYPE")
-            .not(".hidden")
-            .text();
-          var itemHoldText = $J(elem).text();
-          var isMatch =
-            materialText.trim() == "Permanent Reserve" &&
-            itemHoldText.trim() == "Unavailable";
-          if (isMatch) {
-            $J(elem).text("Available");
-          }
-        });
-    });
+  $J(document).ajaxComplete(function() {
+    $J(".detailItems .detailItemTable .detailItemsTableRow")
```

Comment:
Moved this up to go over each item / row rather than the lowest level element. From there it checks each record and doe the work.